ONSdigital/census-rm-ddl
# Updated version of uac-qid migration script.

# Motivation and Context
We have deprecated the IAC Service, but we need to migrate the data out of the Case Processor and into the uacqid schema so that we can guarantee the uniqueness of any UACs and QIDs we generate.

# What has changed
Created migration script.

# How to test?
Set up an environment with the production images of Case Processor and IAC Service. Load an unaddressed batch. Run this script. Check that the data in uacqid.uac_qid matches casev2.uac_qid_link.

# Links
Trello: https://trello.com/c/mlM35xRH
